client-ts/Microsoft.AspNetCore.SignalR.Client.TS.Tests/HubConnection.spec.ts

```diff
@@ -56,7 +56,7 @@ describe("HubConnection", () => {
         it("sends an invocation", async () => {
             let connection = new TestConnection();
 
-            let hubConnection = new HubConnection(connection);
+            let hubConnection = new HubConnection(connection, { logger: null });
```

Comment:
Why do we bother passing in the null logger? We don't need a HubOptionsObject here, right?

Reply:
If the property isn't present, it uses ConsoleLogger by default. In tests, we want no logging at all (because it messes with the test output).